firebase-firestore/src/main/java/com/google/firebase/firestore/FirebaseFirestoreSettings.java

```diff
@@ -221,12 +220,17 @@ public int hashCode() {
   @Override
   @NonNull
   public String toString() {
-    return MoreObjects.toStringHelper(this)
-        .add("host", host)
-        .add("sslEnabled", sslEnabled)
-        .add("persistenceEnabled", persistenceEnabled)
-        .add("timestampsInSnapshotsEnabled", timestampsInSnapshotsEnabled)
-        .toString();
+    StringBuilder builder = new StringBuilder("FirebaseFirestoreSettings{");
+
+    builder.append(String.format("host=%s, ", host));
+    builder.append(String.format("sslEnabled=%s, ", sslEnabled));
+    builder.append(String.format("persistenceEnabled=%s, ", persistenceEnabled));
+    builder.append(
+        String.format("timestampsInSnapshotsEnabled=%s, ", timestampsInSnapshotsEnabled));
+    builder.append(String.format("cacheSizeBytes=%s", cacheSizeBytes));
+
```

Comment:
Nit: This is a somewhat strange combination of StringBuilder and String.format. Do you mind using the pattern used here: https://github.com/firebase/firebase-android-sdk/blob/8409ac490902afb840d5adbe52b86216546921be/firebase-firestore/src/main/java/com/google/firebase/firestore/model/Document.java#L108

Please also take a look here:

https://stackoverflow.com/questions/1532461/stringbuilder-vs-string-concatenation-in-tostring-in-java

TL/DR: The Java compiler will generate the same code for you.

Reply:
Done. Thanks.